engines/dialogs.cpp

```diff
@@ -78,9 +102,10 @@ MainMenuDialog::MainMenuDialog(Engine *engine)
 	GUI::StaticTextWidget *title = new GUI::StaticTextWidget(this, "GlobalMenu.Title", "ScummVM");
 	title->setAlign(Graphics::kTextAlignCenter);
 #endif
-
-	GUI::StaticTextWidget *version = new GUI::StaticTextWidget(this, "GlobalMenu.Version", gScummVMVersionDate);
+     
+	GUI::StaticTextWidget *version = new GUI::StaticTextWidget(this, "GlobalMenu.Version", caption.c_str()) ;
```

Comment:
The description of the task we have on our wiki is "In addition to the ScummVM logo and version, how about showing the engine name and game title at the top of the GMM?" It doesn't say anything about removing the version.

Can you explain why you took the decision to remove the version? Is there is good reason for doing it?